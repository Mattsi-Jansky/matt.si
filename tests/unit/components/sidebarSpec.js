import React from "react"
import renderer from "react-test-renderer"
import Sidebar from "../../../src/components/sidebar"
import siteMetadata from "../../fakes/siteMetadata"

describe("Sidebar", () => {
  it("hasn't changed", () => {
    const sidebar = renderer
      .create(<Sidebar siteMetadata={siteMetadata}/>)
      .toJSON()
    expect(sidebar).toMatchSnapshot()
  })
})
