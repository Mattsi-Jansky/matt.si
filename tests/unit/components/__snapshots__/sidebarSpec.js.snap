// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar hasn't changed 1`] = `
<aside
  className="sidebar"
>
  <header>
    <div
      className="about"
    >
      <div
        className="cover-author-image"
      >
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          <img
            alt="Mattsi Jansky"
            className="author-image"
            src="/pic.jpg"
          />
        </a>
      </div>
      <div
        className="author-name"
      >
        Mattsi Jansky
      </div>
      <p>
        I'm a Software Craftsman at 
        <a
          href="https://codurance.com/"
          target="_blank"
        >
          Codurance
        </a>
        . I love solving problems, learning, and empowering my team.
      </p>
    </div>
  </header>
  <footer>
    <section
      className="contact"
    >
      <hr />
      <ul>
        <li>
          <a
            href="https://twitter.com/mattsijansky"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-twitter"
            />
          </a>
        </li>
        
        <li>
          <a
            href="https://github.com/mattsi-jansky"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-github"
            />
          </a>
        </li>
        <li>
          <a
            href="https://linkedin.com/in/mjjansky"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-linkedin"
            />
          </a>
        </li>
        <li>
          <a
            href="mailto:mattsi@jansky.dev"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-envelope-o"
            />
          </a>
        </li>
        <li>
          <a
            href="/rss.xml"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-rss"
            />
          </a>
        </li>
      </ul>
    </section>
    <div
      className="copyright"
    >
      <p>
        Made with 
        <a
          href="https://www.gatsbyjs.org"
        >
          Gatsby
        </a>
         and
         
        <a
          href="https://github.com/wangonya/flexible-gatsby/"
        >
          Flexible
        </a>
      </p>
    </div>
  </footer>
</aside>
`;
